server/test/test.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-env node, mocha */
```

Comment:
What do you think about moving this into `test/images.js` and requiring it in `test/test.js`. In case at some point we want to test other parts of the site?

Reply:
yea cool.